mantle/kola/tests/misc/network.go

```diff
@@ -209,310 +197,318 @@ func NetworkInitramfsSecondBoot(c cluster.TestCluster) {
 }
 
 var (
-	dhcpClientConfig = `[main]
-          dhcp=dhclient
-`
-
 	captureMacsScript = `#!/usr/bin/env bash
-          set -ex
-          echo "Processing MAC addresses"
-          cmdline=( $(</proc/cmdline) )
-          karg() {
-              local name="$1" value="${2:-}"
-              for arg in "${cmdline[@]}"; do
-                  if [[ "${arg%%=*}" == "${name}" ]]; then
-                      value="${arg#*=}"
-                  fi
-              done
-              echo "${value}"
-          }
-          udevadm settle
-          macs="$(karg macAddressList)"
-          if [[ -z $macs ]]; then
-            echo "No MAC addresses specified."
-            exit 1
-          fi
-          export PRIMARY_MAC=$(echo $macs | awk -F, '{print $1}')
-          export SECONDARY_MAC=$(echo $macs | awk -F, '{print $2}')
-          mount -o rw,remount /boot
-          echo -e "PRIMARY_MAC=${PRIMARY_MAC}\nSECONDARY_MAC=${SECONDARY_MAC}" > /boot/mac_addresses
+	set -ex
+	echo "Processing MAC addresses"
+	cmdline=( $(</proc/cmdline) )
+	karg() {
+		local name="$1" value="${2:-}"
+		for arg in "${cmdline[@]}"; do
+			if [[ "${arg%%=*}" == "${name}" ]]; then
+				value="${arg#*=}"
+			fi
+		done
+		echo "${value}"
+	}
+	# Wait for device nodes
+	udevadm settle
+
+	macs="$(karg macAddressList)"
+	if [[ -z $macs ]]; then
+		echo "No MAC addresses specified."
+		exit 0
+	fi
+
+	export PRIMARY_MAC=$(echo $macs | awk -F, '{print $1}')
+	export SECONDARY_MAC=$(echo $macs | awk -F, '{print $2}')
+	mount -o rw,remount /boot
+	echo -e "PRIMARY_MAC=${PRIMARY_MAC}\nSECONDARY_MAC=${SECONDARY_MAC}" > /boot/mac_addresses
+	`
+
+	initInterfacesScript = `#!/usr/bin/env bash
```

Comment:
I wonder if we should link to https://github.com/RHsyseng/rhcos-slb/blob/161a421f8fdcdb4b08fb6366daa8fe1b75cbe095/init-interfaces.sh again here to make it clear this is copy pasted verbatim.

Reply:
It is indeed copy-pasted. 
I would find it easier to just curl the script directly from our repo, but I'm not sure you'll want to connect our repos in such a strong connection. 
How about adding the link in a comment then?

Reply:
> How about adding the link in a comment then?

DONE